### DISCUSSION TITLE
Why GPL?

### DISCUSSION THREAD

**Discussion by guettli** [libgit2/pygit2#1085] [Q&A] [answered]
First I thougt "pygit2 looks good". Then I saw the license ... :-(

I must not use GPL software.

Why did you choose this license?

**Comment by jdavid** [comment, marked as answer]
GPLv2 _with linking exception_

pygit2 uses the same license as libgit2.

So why libgit2 uses this license? Quoting @vmg :

> it uses code sourced from Git, whose authors requested we maintain the GPL and allowed us to add a linking exception.

**Comment by kuwv** [comment]
@guettli https://en.wikipedia.org/wiki/GPL_linking_exception
